server/controllers/commands/webhook.js

```diff
@@ -1,7 +1,7 @@
-const Channel = require('../models').Channel;
+const Channel = require('../../models').Channel;
 
-const slack = require('../api/slack')
-const webhookHelper = require('../helpers/webhook')
+const slack = require('../../api/slack')
+const webhookHelper = require('../../helpers/webhook')
```

Comment:
I originally created the `server/commands/` directory in https://github.com/datadotworld/slack-app/pull/43. Since then, I've started to feel like `commands/` didn't make sense as a top-level directory since all of the commands would be used only in the controllers. This is just moving this directory as a sub-directory under `controllers/`.